modules/gapi/include/opencv2/gapi/garg.hpp

```diff
@@ -235,8 +235,50 @@ inline GRunArgsP& operator += (GRunArgsP &lhs, const GRunArgsP &rhs)
 
 namespace gapi
 {
+    /**
+     * \addtogroup gapi_serialization
+     * @{
+     *
+     * @brief G-API functions and classes for serialization and deserialization.
+     */
+    /** @brief Binds deserialized output GRunArgs to GRunArgsP which can be used by GCompiled.
```

Comment:
I understand this but please explain more on the case when it is actually required.

Reply:
For example, what "binging" means here? How a `GRunArg` (value) turns into `GRunArgP` (a pointer to value)? Why do we need this?

Reply:
Added a little explanation